connector/kafka-0-10-sql/src/main/resources/error/kafka-error-conditions.json

```diff
@@ -97,5 +104,21 @@
       }
     },
     "sqlState" : "22000"
+  },
+  "RESOLVED_START_OFFSET_GREATER_THAN_END_OFFSET" : {
+    "message" : [
+      "The resolved start offset <startOffset> is greater than the resolved end offset <endOffset> for topic <topic> partition <partition>.",
+      "There may be multiple reasons:",
```

Comment:
Same - I'd recommend repharasing the sentence a bit. This is user facing, so we should make sure moderate users could understand.

I need to think about it a bit more - it'd be awesome if you can come up with better refined message before I come up with alternative.

Reply:
Also, probably ideal if we can think more about how we can guide what to check and what to do from user end, instead of explaining what condition could trigger this.